jib-core/src/main/java/com/google/cloud/tools/jib/api/ImageReference.java

```diff
@@ -334,18 +360,27 @@ public String toString() {
     // Use tag if not the default tag.
     if (!DEFAULT_TAG.equals(tag)) {
       // Append with "@tag" instead of ":tag" if tag is a digest
```

Comment:
Remove this comment.

BTW, I think it is correct that this method prints both a (non-default) tag and a digest.